Source/WebCore/svg/SVGLengthValue.h

```diff
@@ -79,7 +80,7 @@ class SVGLengthValue {
     SVGLengthMode lengthMode() const { return m_lengthMode; }
 
     bool isZero() const { return !m_valueInSpecifiedUnits;  }
-    bool isRelative() const { return m_lengthType == SVGLengthType::Percentage || m_lengthType == SVGLengthType::Ems || m_lengthType == SVGLengthType::Exs || m_lengthType == SVGLengthType::Ch; }
+    bool isRelative() const { return m_lengthType == SVGLengthType::Percentage || m_lengthType == SVGLengthType::Ems || m_lengthType == SVGLengthType::Exs || m_lengthType == SVGLengthType::Ch || m_lengthType == SVGLengthType::Rems; }
```

Comment:
It would be interesting to test how `rem` behaves in other browsers with regards to sizes. Simple SVG test cases should show the differences in the final pixel sizes. 

`isRelative` is used at a number of places
https://searchfox.org/wubkat/search?q=symbol:_ZNK7WebCore14SVGLengthValue10isRelativeEv&redirect=false